packages/custom-functions-metadata/src/generate.ts

```diff
@@ -739,9 +739,9 @@ function findTag(node: ts.Node, tagName: string): ts.JSDocTag | undefined {
 /**
  * If a node contains the named tag, returns the tag comment, otherwise returns "".
  */
-function getTagComment(node: ts.Node, tagName: string) {
+function getTagComment(node: ts.Node, tagName: string): string {
   const tag = findTag(node, tagName);
-  return tag && tag.comment ? tag.comment : "";
```

Comment:
Could this be updated to `return tag?.comment?.toString() || ""` ?  Do you think it is better that way?